vnext/Microsoft.ReactNative.IntegrationTests/ReactNativeHostTests.cpp

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 #include "pch.h"
+#include <JsiAbiApi.h>
```

Comment:
[nit] Is there any code here that needs the decls in this header?

Reply:
No, it is just a way for me to get the code compiled. I will add a true test for the JSI ABI and remove this include from this file.

---
In reply to: [451862528](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r451862528) [](ancestors = 451862528)